CMSgov/mdct-mcr
# Update parsing.ts

## Description

Error being thrown when choices do not have children

### How to test


Will need to test in `Val` to be sure

- Log in to val as stateuser1
- Select program Test23
- Export pdf
- In /export tab, open console to see error. See attached image also.

See if this pops up: 
![Screen Shot 2023-01-13 at 10 09 27 AM](https://user-images.githubusercontent.com/8299237/212366123-0b874c9f-4846-4f94-a274-7d45c801360d.png)


### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
